spec/features/ncr/work_orders/post_approval_modification_spec.rb

```diff
@@ -82,7 +82,10 @@ def expect_budget_approvals_restarted(work_order)
     ))
 
     approver = work_order.budget_approvers.first
+    expect(approver.email_address).to eq(Ncr::Mailboxes.ba61_tier1_budget.email_address)
     reapproval_mail = deliveries.find { |mail| mail.to.include?(approver.email_address) }
     expect(reapproval_mail.html_part.body).to include('Approve')
+    approver_page_row = page.find(:css, ".step-row.pending.position-1")
```

Comment:
That's very neat. I was looking for something like that but didn't immediately see where it was being used. Any pointers?